websites/tasks.py

```diff
@@ -0,0 +1,68 @@
+""" Tasks for websites """
+import logging
+
+import boto3
+import celery
+from django.conf import settings
+
+from main.celery import app
+from main.utils import chunks
+from websites.api import import_ocw2hugo_course
+
+log = logging.getLogger(__name__)
+
+
+@app.task(bind=True)
```

Comment:
`self` isn't used so this doesn't need `bind=True`, which will also let you remove `# pylint:disable=unused-argument`